applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -460,20 +458,8 @@ def _SetPhysicalProperties(self):
         return materials_imported
 
     def _SetNodalProperties(self):
-        # Get density and dynamic viscostity from the properties of the first element
-        for el in self.main_model_part.Elements:
-            rho = el.Properties.GetValue(KratosMultiphysics.DENSITY)
-            if rho <= 0.0:
-                raise Exception("DENSITY set to {0} in Properties {1}, positive number expected.".format(rho,el.Properties.Id))
-            dyn_viscosity = el.Properties.GetValue(KratosMultiphysics.DYNAMIC_VISCOSITY)
-            if dyn_viscosity <= 0.0:
-                raise Exception("DYNAMIC_VISCOSITY set to {0} in Properties {1}, positive number expected.".format(dyn_viscosity,el.Properties.Id))
-            break
-        else:
-            raise Exception("No fluid elements found in the main model part.")
-        # Transfer the obtained properties to the nodes
-        KratosMultiphysics.VariableUtils().SetVariable(KratosMultiphysics.DENSITY, rho, self.main_model_part.Nodes)
-        KratosMultiphysics.VariableUtils().SetVariable(KratosMultiphysics.DYNAMIC_VISCOSITY, dyn_viscosity, self.main_model_part.Nodes)
+        # This is required to avoid the base solver to set the DENSITY and DYNAMIC_VISCOSITY in the nodes
+        pass
```

Comment:
I would remove this, and the copy base class implementation to navier_stokes_solver_vmsmonolithic, navier_stokes_solver_fractional_step ones. We only need this on those two ryt? So once we finally agree to change old VMS and FS elements, we can get rid of these methods as well without changing embedded solvers ryt?

Reply:
I did it in that way to avoid changes in the solvers as much as possible. Nevertheless, I agree this is the proper way to do it. I'll change it :+1:

Reply:
Done.